tests/unit/test_sandbox.py

```diff
@@ -19,18 +19,6 @@ def temp_dir(monkeypatch):
         yield temp_dir
 
 
-def test_env_vars(temp_dir):
```

Comment:
Imho, this test is crucial, I found out in my own branch when working on all the initialization methods.

Reply:
Because we refractor to move the `add_env_var` method to `Runtime`, so we have to test them there. But we do include relevant env var related tests in `test_runtime.py` so this should still be tested!